test/1_test_fullpath.js

```diff
@@ -340,37 +346,43 @@ contract('Voucher tests', async (addresses) => {
 
       //Check BosonRouter state
       assert.equal(
-        await contractBosonRouter.correlationIds(users.seller.address),
+        await contractBosonRouter.getCorrelationId(users.seller.address),
         2,
         'Correlation Id incorrect'
       );
 
       //Check VocherKernel State
-      const promise = await contractVoucherKernel.promises(promiseId2);
-      assert.strictEqual(promise.promiseId, promiseId2, 'Promise Id incorrect');
-      assert.isTrue(promise.nonce.eq(new BN(2)));
+      const promiseData = await contractVoucherKernel.getPromiseData(
+        promiseId2
+      );
+      assert.equal(promiseData[0], promiseId2, 'Promise Id incorrect');
```

Comment:
Please use constants instead of hard-coded indexes for promiseData (same as remark above)